nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -509,7 +509,7 @@ def get_statistic_points(self, model: TModel, graph: NNCFGraph) -> StatisticPoin
                 TargetType.POST_LAYER_OPERATION, node_name, port_id=OUTPUT_PORT_OF_NODE
             )
             stat_collector = self._backend_entity.mean_statistic_collector(
-                reduction_shape=channel_axis, num_samples=self.subset_size, inplace=self.inplace_statistics
+                reduction_axes=channel_axis, num_samples=self.subset_size, inplace=self.inplace_statistics
```

Comment:
`reduction_axes` should be `Tuple`, while `channel_axis` represented as `int`.

Reply:
Good catch, done